extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/videos/GoogleVideosImporter.java

```diff
@@ -127,27 +122,26 @@ void importSingleVideo(
     }
 
     String uploadToken =
-            getOrCreateVideosInterface(authData).uploadVideoContent(inputStream, filename);
+        getOrCreateVideosInterface(authData).uploadVideoContent(inputStream, filename);
 
     NewMediaItem newMediaItem = new NewMediaItem(filename, uploadToken);
 
     NewMediaItemUpload uploadItem =
-            new NewMediaItemUpload(null, Collections.singletonList(newMediaItem));
+        new NewMediaItemUpload(null, Collections.singletonList(newMediaItem));
 
-    executor.executeAndSwallowIOExceptions(
-            inputVideo.getDataId(),
-            inputVideo.getName(),
-            () -> getOrCreateVideosInterface(authData).createVideo(uploadItem));
+    final NewMediaItemResult result =
+        getOrCreateVideosInterface(authData).createVideo(uploadItem).getResults()[0];
```

Comment:
Do we want to ensure the result array is not empty?

Reply:
The GooglePhotosImporter makes the same assumption and we do check for success within the VideosInterface. Might make a separate fix which handles both the Videos and the Photos Importer.